src/config/media.jsx

```diff
@@ -0,0 +1,14 @@
+export default [
+	{
+		date: 'June 16, 2015',
+		description: 'The Daily Bruin wrote an article about ACM',
+	},
+	{
+		date: 'December 8, 2016',
+		description: 'This paper featured a hackathon run by ACM',
+	},
+	{
+		date: 'April 20, 2018',
+		description: 'ACM is so popular!',
+	}
+]
```

Comment:
Looks good! I like that you came up with some nice descriptions of possible media posts!